doc/release-notes.md

```diff
@@ -70,161 +70,155 @@ New RPCs
 - `getbalances` returns an object with all balances (`mine`,
   `untrusted_pending` and `immature`). Please refer to the RPC help of
   `getbalances` for details. The new RPC is intended to replace
-  `getunconfirmedbalance` and the balance fields in `getwalletinfo`, as well as
-  `getbalance`. The old calls may be removed in a future version.
+  `getbalance`, `getunconfirmedbalance`, and the balance fields in
+  `getwalletinfo`.  These old calls and fields may be removed in a
+  future version. (#15930, #16239)
 
-- A new `setwalletflag` RPC sets/unsets flags for an existing wallet.
+- `setwalletflag` sets and unsets wallet flags that enable or disable
+  features specific to that existing wallet, such as the new
+  `avoid_reuse` feature documented elsewhere in these release notes.
+  (#13756)
 
 Updated RPCs
 ------------
 
 Note: some low-level RPC changes mainly useful for testing are described in the
 Low-level Changes section below.
 
-- The `sendmany` RPC had an argument `minconf` that was not well specified and
-  would lead to RPC errors even when the wallet's coin selection would succeed.
-  The `sendtoaddress` RPC never had this check, so to normalize the behavior,
-  `minconf` is now ignored in `sendmany`. If the coin selection does not
-  succeed due to missing coins, it will still throw an RPC error. Be reminded
-  that coin selection is influenced by the `-spendzeroconfchange`,
-  `-limitancestorcount`, `-limitdescendantcount` and `-walletrejectlongchains`
-  command line arguments.
-
-- Several RPCs have been updated to include an "avoid_reuse" flag, used
-  to control whether already used addresses should be left out or
-  included in the operation.  These include:
-
-    - createwallet
-    - getbalance
-    - getbalances
-    - sendtoaddress
-
-  In addition, `sendtoaddress` has been changed to avoid partial spends
-  when `avoid_reuse` is enabled (if not already enabled via the
-  `-avoidpartialspends` command line flag), as it would otherwise risk
-  using up the "wrong" UTXO for an address reuse case.
-
-  The listunspent RPC has also been updated to now include a "reused"
-  bool, for nodes with "avoid_reuse" enabled.
-
-- The `getblockstats` RPC is faster for fee calculation by using
-  BlockUndo data. Also , `-txindex` is no longer required and
-  `getblockstats` works for all non-pruned blocks.
-
-- `createwallet` can now create encrypted wallets if a non-empty
-  passphrase is specified.
-
-- The `utxoupdatepsbt` RPC method has been updated to take a
-  `descriptors` argument. When provided, input and output scripts and
-  keys will be filled in when known, and P2SH-witness inputs will be
-  filled in from the UTXO set when a descriptor is provided that shows
-  they're spending segwit outputs.
-
-  See the RPC help text for full details.
-
-- The -maxtxfee setting no longer has any effect on non-wallet RPCs.
```

Comment:
note: this was also dropped (seems ok though)

Reply:
This was just substantially rewritten.  Original was:

```markdown
- The -maxtxfee setting no longer has any effect on non-wallet RPCs.

  The `sendrawtransaction` and `testmempoolaccept` RPC methods previously
  accepted an `allowhighfees` parameter to fail the mempool acceptance in case
  the transaction's fee would exceed the value of the command line argument
  `-maxtxfee`. To uncouple the RPCs from the global option, they now have a
  hardcoded default for the maximum transaction fee, that can be changed for
  both RPCs on a per-call basis with the `maxfeerate` parameter. The
  `allowhighfees` boolean option has been removed and replaced by the
  `maxfeerate` numeric option.
```

Revised is:

```markdown
- `sendrawtransaction` and `testmempoolaccept` no longer accept a
  `allowhighfees` parameter to fail mempool acceptance if the
  transaction fee exceedes the value of the configuration option
  `-maxtxfee`.  Now there is a hardcoded default maximum feerate that
  can be changed when calling either RPC using a `maxfeerate` parameter.
  (#15620)
```

The other note you identified as lost does seem to be so.  I'll create a followup PR for that and the formatting change.  Thanks!